static/js/pv_tree.tsx

```diff
@@ -0,0 +1,169 @@
+import React, { Component } from "react";
+import hierarchy from "../../tools/pv_tree_generator/hierarchy_golden.json";
+//import hierarchy from "../data/hierarchy.json";
+
+interface NodePropType {
+  title: string;
+  selected: string;
+  expanded: string;
+  count: number;
+  children: NodePropType[];
+  type: string;
+  argString: string;
+  updateUrl: (statvar: string, add: boolean) => void;
+  nodePath: string;
+}
+
+interface NodeStateType {
+  checked: boolean;
+  expanded: boolean;
+  nodePath: string;
+}
+
+class Node extends Component<NodePropType, NodeStateType> {
+  constructor(props: NodePropType) {
+    super(props);
+    this._handleCheckboxClick = this._handleCheckboxClick.bind(this);
+    this._handleExpandClick = this._handleExpandClick.bind(this);
+    this.state = {
+      checked: props.selected === "yes" ? true : false,
+      expanded: props.selected === "yes" ? true : false,
+      nodePath: props.nodePath + "," + props.title.replace(/\s/g, ""),
+    };
+  }
+
+  public render = (): JSX.Element => {
+    let checkboxImg: JSX.Element;
+    let expandImg: JSX.Element;
+    let child: JSX.Element[];
+    if (this.props.type === "value") {
+      checkboxImg = (
+        <button
+          className={this.state.checked ? "checkbox checked" : "checkbox"}
+          onClick={this._handleCheckboxClick}
+        />
+      );
+    }
+
+    if (this.state.expanded) {
+      expandImg = (
+        <img
+          className="right-caret transform-down"
+          src="../images/right-caret.png"
+          width="12px"
```

Comment:
Could you move this parameter to the stylesheet? Looks like if we apply it to the right-caret class, it wouldn't be needed below on line 80 as well.

Reply:
done.